.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  push:
+    branches: [main]
+
+name: Release
+
+permissions:
+  contents: write
+
+env:
+  CARGO_ARGS: --no-default-features --features stdlib,zlib,importlib,encodings,sqlite,ssl
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+
+      - name: Set up Windows Environment
+        shell: bash
+        run: |
+          cargo install --target-dir=target -v cargo-vcpkg
+          cargo vcpkg -v build
+        if: runner.os == 'Windows'
+      - name: Set up MacOS Environment
+        run: brew install autoconf automake libtool
+        if: runner.os == 'macOS'
+
+      - name: Build RustPython
+        run: cargo build --release --verbose --features=threading ${{ env.CARGO_ARGS }}
+        if: runner.os == 'macOS'
+      - name: Build RustPython
+        run: cargo build --release --verbose --features=threading ${{ env.CARGO_ARGS }},jit
+        if: runner.os != 'macOS'
+
+      - name: Rename Binary
+        run: cp target/release/rustpython target/release/rustpython-release-${{ matrix.os }}
```

Comment:
The OS name is not enough to identify binary target. Could you please also add architecture names to the binary too?

e.g. macOS version can be either x86_64 or aarch64 unless universal build. Linux version can be a lot more variant. Windows version can be x86, x86_64 or arm64.

Here is some platform/architecture matrix
https://github.com/youknowone/python-deadlib/blob/main/.github/workflows/_crypt.yml#L20-L37

Reply:
Good catch! Overlooked this in haste

Here's the new naming: https://github.com/theshubhamp/RustPython/releases/tag/2024-12-11-gh-rel-rustpython-15
![image](https://github.com/user-attachments/assets/de1ad110-43f6-4c89-9d49-e7624d6b73f8)

And the corresponding file types:
![image](https://github.com/user-attachments/assets/1e31e7cc-e525-46df-b88d-18755f130e40)

I added other targets too (commented out), but they need some extra work to be fully cross compile-able. Fine to do that as a follow up PR ?